addons/website_slides/views/res_config_settings_views.xml

```diff
@@ -29,55 +30,34 @@
             </xpath>
             <xpath expr="//div[hasclass('settings')]" position="inside">
                 <div class="app_settings_block" data-string="eLearning" string="eLearning" data-key="website_slides">
-                    <h2>eLearning</h2>
-                    <div class="row mt16 o_settings_container" id="elearning_selection_settings">
-                        <div class="col-12 col-lg-6 o_setting_box" id="elearning_install_forum">
-                            <div class="o_setting_left_pane">
-                                <field name="module_website_slides_forum"/>
-                            </div>
-                            <div class="o_setting_right_pane">
-                                <label for="module_website_slides_forum"/>
-                                <div class="text-muted">
-                                    Create a community and let the members help each others
+                        <h2>eLearning</h2>
+                        <div class="row mt16 o_settings_container" id="elearning_selection_settings">
```

Comment:
I would use `website_slides_install_mass_mailing_slides` to be consistent with other ids in the same view